paddle/fluid/pir/dialect/op_generator/op_gen.py

```diff
@@ -349,53 +351,48 @@ class {TEST_API} {op_name} : public pir::Op<{op_name}{interfaces}{traits}> {{
     'expand',
 }
 
+OpNamePair = namedtuple('OpNamePair', ['phi_name', 'fluid_name'])
 
-def to_phi_and_fluid_op_name(op_item):
+
+def to_phi_and_fluid_op_name(op_item: str) -> OpNamePair:
     # Template: - op : phi_name (fluid_name)
     names = op_item.split('(')
     if len(names) == 1:
         phi_fluid_name = names[0].strip()
-        return phi_fluid_name, phi_fluid_name
+        return OpNamePair(phi_fluid_name, phi_fluid_name)
     else:
         phi_name = names[0].strip()
         fluid_name = names[1].split(')')[0].strip()
-        return phi_name, fluid_name
+        return OpNamePair(phi_name, fluid_name)
 
 
-def to_phi_and_fluid_grad_op_name(op_item):
+def to_phi_and_fluid_grad_op_name(op_item: str) -> List[OpNamePair]:
     # Template: sum_grad (reduce_sum_grad), sum_double_grad
-    rtn = []
-    all_names = op_item.split(', ')
-    for name in all_names:
-        backward_phi_name, backward_fluid_name = to_phi_and_fluid_op_name(name)
-        rtn.append([backward_phi_name, backward_fluid_name])
-    return rtn
+    return map(to_phi_and_fluid_op_name, op_item.split(', '))
 
 
 # =====================================
 # Parse Op Compat From Yaml
 # =====================================
 class OpCompatParser:
-    def __init__(self, ops_compat_yaml_file):
+    def __init__(self, ops_compat_yaml_file: str):
         self.ops_compat_yaml_file = ops_compat_yaml_file
         with open(self.ops_compat_yaml_file, "r") as f:
             self.ops_compat = yaml.safe_load(f)
 
-    def get_compat(self, op_name):
+    def get_compat(self, op_name: str):
         for compat in self.ops_compat:
-            forward_phi_name, forward_fluid_name = to_phi_and_fluid_op_name(
-                compat['op']
-            )
-            if op_name == forward_phi_name:
+            name_pair = to_phi_and_fluid_op_name(compat['op'])
+            if op_name == name_pair.phi_name:
                 return compat
             elif 'backward' in compat.keys():
                 bkw_names = to_phi_and_fluid_grad_op_name(compat['backward'])
                 for name in bkw_names:
-                    if op_name == name[0]:
+                    if op_name == name.phi_name:
                         return compat
         return None
 
-    def parse_support_tensor(self, op):
+    def parse_support_tensor(self, op) -> Tuple[Dict, Dict]:
```

Comment:
```suggestion
    def parse_support_tensor(self, op) -> Tuple[Dict, Dict]:
```

dict 的泛型参数不要省略吧，就算是 `dict[Any, Any]` 也写出来

Reply:
Done